<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDlgAppSettings</class>
 <widget class="QDialog" name="CDlgAppSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>312</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>312</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>312</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>243</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>16</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>16</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab_1">
       <attribute name="title">
        <string>Grid Layout</string>
       </attribute>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>97</y>
          <width>52</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>color</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="eGridColor">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>90</y>
          <width>113</width>
          <height>25</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>hhhhhh</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="maxLength">
         <number>6</number>
        </property>
       </widget>
       <widget class="QCheckBox" name="cShowGrid">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>17</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>show grid</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>59</y>
          <width>52</width>
          <height>15</height>
         </rect>
        </property>
        <property name="text">
         <string>grid size</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbGridSize">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>53</y>
          <width>63</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="btnGridColor">
        <property name="geometry">
         <rect>
          <x>203</x>
          <y>90</y>
          <width>71</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Shortcuts</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>381</width>
          <height>211</height>
         </rect>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <column/>
        <column/>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Test Level</string>
       </attribute>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>44</y>
          <width>52</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Score</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbSkill">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>8</y>
          <width>251</width>
          <height>25</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>52</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Lives</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbLives">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>52</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Hit Points</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="eHitPoints">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>70</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>000</string>
        </property>
        <property name="placeholderText">
         <string>50</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>15</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Select Skill Level</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="eScore">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>40</y>
          <width>113</width>
          <height>22</height>
         </rect>
        </property>
        <property name="inputMask">
         <string>0000000</string>
        </property>
        <property name="placeholderText">
         <string>0</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Updater</string>
       </attribute>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>65</y>
          <width>52</width>
          <height>14</height>
         </rect>
        </property>
        <property name="text">
         <string>Api url</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="eApiURL">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>60</y>
          <width>291</width>
          <height>24</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="cCheckUpdate">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>321</width>
          <height>19</height>
         </rect>
        </property>
        <property name="text">
         <string>Check for newer verrsions when start-up.</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCheckUpdate">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>170</y>
          <width>85</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Check now</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Editor</string>
       </attribute>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>15</y>
          <width>62</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Font size</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cbFontSize">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>8</y>
          <width>85</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
        <property name="centerButtons">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cShowGrid</tabstop>
  <tabstop>cbGridSize</tabstop>
  <tabstop>eGridColor</tabstop>
  <tabstop>btnGridColor</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>cbSkill</tabstop>
  <tabstop>eScore</tabstop>
  <tabstop>eHitPoints</tabstop>
  <tabstop>cbLives</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDlgAppSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDlgAppSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
